nextjs_vers/layout/header/nav/link.tsx

```diff
@@ -5,6 +5,8 @@ import style from './nav.module.scss'
 
 
 // A component to extend the functionality of the default 'Link' component - can't add an active className to the default one
+
+
 const CustomNavLink = ({ href, children }) => {
```

Comment:
Add interface,

if cant set activeClassName then create a function to check if link is active using nextjs hook useRouter

Reply:
Never mind lol just seen useRouter is used here